Sources/BrowserServicesKit/ContentBlocking/UserScripts/surrogates.js

```diff
@@ -504,9 +504,17 @@
         // Tracker blocking is dealt with by content rules
         // Only handle surrogates here
         if (blocked && isSurrogate && !isTrackerAllowlisted(topLevelUrl, trackerUrl)) {
+            // Remove error handlers on the original element
+            if (element && element.onerror) {
```

Comment:
@alistairjcbrown So this improves the test case: https://privacy-test-pages.glitch.me/privacy-protections/surrogates/ I know we talked about it and suggested it shouldn't make a difference but it does, I think this is still fragile given the "load" event fires after the page loads and the mutation observers currently are delayed a little too.